packages/node/test/integrations/http.test.ts

```diff
@@ -133,6 +134,23 @@ describe('tracing', () => {
     );
   });
 
+  it('does not add the user_id to the baggage header if sendDefaultPii is set to false', async () => {
+    nock('http://dogs.are.great').get('/').reply(200);
+
+    createTransactionOnScope(false);
```

Comment:
Just seeing this in tests is so glorius. `Sentry.startTransaction()` should really just instantly put a transaction on the scope...